content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md

```diff
@@ -0,0 +1,235 @@
+- This command is not meant to be a full, feature-rich query tool.
+  It's meant for debug, triage, and basic data exploration.
+{{% /note %}}
+
+### InfluxDB connection configuration
+
+Your {{< product-name omit=" Clustered" >}} cluster host and port should be
+configured in your in your `influxctl`
+[connection profile](/influxdb/cloud-dedicated/reference/cli/influxctl/#configure-connection-profiles).
+By default TLS and port 443 are used.
+You can also set a default database and token to use for the `query` and `write`
+commands in your connection profile, but these can also be passed with the
+command using the `--database` and `--token` flags.
+Command line flags override settings in the connection profile. 
+
+### Output format
+
+The `--format` option lets you print the output in other formats.
+By default, the 'table' format is used, but the 'json' format is
+available for programmatic parsing by other tooling.
+
+## Usage
+
+```sh
+influxctl query [flags] <QUERY>
+```
+
+## Arguments
+
+| Argument  | Description                                                                         |
+| :-------- | :---------------------------------------------------------------------------------- |
+| **QUERY** | SQL query to execute (command line string, path to file, or `-` to read from stdin) |
+
+## Flags
+
+| Flag |              | Description                                                  |
+| :--- | :----------- | :----------------------------------------------------------- |
+|      | `--database` | Database to query                                            |
+|      | `--format`   | Output format (`table` _(default)_ or `json`)                |
+|      | `--token`    | Database token with read permissions on the queried database |
+| `-h` | `--help`     | Output command help                                          |
+
+{{% caption %}}
+_Also see [`influxctl` global flags](/influxdb/cloud-dedicated/reference/cli/influxctl/#global-flags)._
+{{% /caption %}}
+
+## Examples
+
+- [Query InfluxDB v3 and return results in table format](#query-influxdb-v3-and-return-results-in-table-format)
+- [Query InfluxDB v3 and return results in JSON format](#query-influxdb-v3-and-return-results-in-json-format)
+
+In the examples below, replace the following:
+
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}:
+  Database token with read access to the queried database
+- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:
+  Name of the database to query
+
+### Query InfluxDB v3 and return results in table format
+
+{{% code-placeholders "DATABASE_(TOKEN|NAME)" %}}
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[string](#)
+[file](#)
+[stdin](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+{{% influxdb/custom-timestamps %}}
+```sh
+influxctl query \
+  --token DATABASE_TOKEN \
+  --database DATABASE_NAME \
+  "SELECT * FROM home WHERE time >= '2022-01-01T08:00:00Z' LIMIT 5"
+```
```

Comment:
Is it worth showing one example without the token and database flags?

Reply:
Sure, I'll add one.